sdk/src/types/block/context_input/block_issuance_credit.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use derive_more::{Display, From};
+
+use crate::types::block::output::AccountId;
+
+/// A Block Issuance Credit Input provides the VM with context for the value of
+/// the BIC vector of a specific slot.
```

Comment:
Can we write out BIC?

Reply:
or maybe rephrase to avoid repetition, I dunno. Maybe put (BIC) after the first one.